CMSgov/mint-app
# [EASI-2492] - Launch Darkly Flags to Hide IT Lead experience

# EASI-2492

## Changes and Description

- Updated `launchdarkly-react-client-sdk`
- Added Launch Darkly Flag `hideITLeadExperience` for all 4 environments in LD
- Removed old EASI flags in MINT and replaced with `hideITLeadExperience`
- Added flag conditional on task list component and `it-solutions` route
- Added BE support for future reading flag from file for CI cypress testing

## Notes

- Was having some issues with getting flag from files config to wirk with CI. This work was not needed part of the PR but in preparation for future cypress config, carried over from EASI.  Commented out file flag config for now.

## How to test this change

- Attempt to enter IT Solutions from the task list and verify it isn't visible
- Attempt to enter TI Solutions via url 'http://localhost:3005/models/:modelPlanID/task-list/it-solutions'

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
